pubspec.yaml

```diff
@@ -0,0 +1,12 @@
+name: browser_launcher
+description: A standardized way to launch web browsers.
+
+version: 0.1.0-dev
+
+author: Dart Team <misc@dartlang.org>
+homepage: https://github.com/dart-lang/browser_launcher
+
+environment:
+  sdk: '>=2.2.0-dev <3.0.0'
+
+dependencies:
```

Comment:
Need to add `dev_dependnecies` + pedantic. 

See https://github.com/dart-lang/shelf/blob/master/pubspec.yaml#L21